robot/lp/communication.robot

```diff
@@ -0,0 +1,62 @@
+*** Settings ***
+Documentation       A test suite for the length previx framing communication
+
+Library             Process
+Library             String
+Library             OperatingSystem
+
+Test Setup          Initialize LP Chat Server
+Test Teardown       Terminate LP Chat Server
+
+
+*** Variables ***
+${SERVER_HOST}      localhost
+${SERVER_PORT}      55519
+${SERVER_PATH}      /path/to/the/server
+${CLIENT_PATH}      /path/to/the/client
+
+${INPUT}            Hello\nToday is a wonderful day!\n
+${BASELINE}         bob: Hello\nbob: Today is a wonderful day!
+
+
+*** Test Cases ***
+Send Some message
+    Log to console    START FIRST PROCESS
```

Comment:
```suggestion
```

Looks like debugging leftovers. We already have the regular Robot logs.

Reply:
Ah, right. Pipeline is red. However, I'll just expose the Robot logs as artifacts instead.